.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -20,6 +20,15 @@ body:
   validations:
     required: true
 
+- type: textarea
+  attributes:
+    label: Issue Map
+    description: A map that the issue occurs on. 
+    placeholder: |
+      Please explicitly state what map the issue occurs on. If this is not a default map, include a download link to the BSP, VMF, or both.
+  validations:
+    required: true
```

Comment:
What happens if it's not a map bug?

Reply:
Good catch, though im kind of inclined to leave it this way and just have people occasionally put "n/a" rather than make it optional. making it optional just means we'll be in the same spot we are now of hunting people down.